refarch-backend/pom.xml

```diff
@@ -439,6 +448,21 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <!-- to easily reset/clean local stack database: "mvn flyway:clean flyway:migrate -Dflyway.cleanDisabled=false" -->
+                <groupId>org.flywaydb</groupId>
+                <artifactId>flyway-maven-plugin</artifactId>
+                <configuration>
+                    <url>jdbc:postgresql://localhost:5432/postgres</url>
+                    <user>admin</user>
+                    <password>admin</password>
```

Comment:
It looks like the plugin even supports specific maven properties out of the box so we could move those parts up to the properties section to make configuration even easier when different environment is desired. See https://documentation.red-gate.com/fd/maven-goal-184127408.html#MavenGoal-Mavenproperties

Reply:
done